File: app/src/main/java/gardezi/io/amiibofinder/data/database/AppDatabase.java
package gardezi.io.amiibofinder.data.database;

import android.arch.persistence.room.Database;
import android.arch.persistence.room.RoomDatabase;

import gardezi.io.amiibofinder.data.database.dao.AmiiboDAO;
import gardezi.io.amiibofinder.model.Amiibo;

@Database(entities = {Amiibo.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {

    public abstract AmiiboDAO amiiboDAO();

}

File: app/src/main/java/gardezi/io/amiibofinder/data/database/AppDatabase.kt
package gardezi.io.amiibofinder.data.database

import android.arch.persistence.room.Database
import android.arch.persistence.room.RoomDatabase

import gardezi.io.amiibofinder.data.database.dao.AmiiboDAO
import gardezi.io.amiibofinder.model.Amiibo

@Database(entities = arrayOf(Amiibo::class), version = 1)
abstract class AppDatabase : RoomDatabase() {

    abstract fun amiiboDAO(): AmiiboDAO

}
